Tests/FunctionalTests/ResourcesTests.swift

```diff
@@ -114,6 +114,15 @@ class ResourcesTests: XCTestCase {
         }
     }
 
+    func testSwiftResourceAccessorDoesNotCauseInconsistentImportWarning() throws {
+        try fixture(name: "Resources/FoundationlessClient/UtilsWithFoundationPkg") { fixturePath in
+            XCTAssertBuilds(
+                fixturePath,
+                Xswiftc: ["-warnings-as-errors"]
```

Comment:
Starting a thread to discuss...

> I'm not sure this warning should be triggered at all when the imports do not match exactly... Or, we should have some way to silence it in code, defaulting to parens.

I did some researching and there is some good discussion in https://forums.swift.org/t/update-on-implementation-only-imports/26996/12 as to why the warning is triggered in the first place. What makes this tricky is that I do not think there is a way to pass a flag to silence only this warning.

Reply:
Understand the rationale, but this issue points to a use case where the current design doesn’t serve our needs (and, incidentally, if it were ever stabilized as an “access level” on `import` like once pitched, wouldn’t behave like access modifiers do elsewhere).

I posted some thoughts over in the linked PR about why even import scanning would be brittle, and I think possibly the most resilient solution here is to come up with a new “flavor” of `@_implementationOnly` (rather than a flag) that explicitly acknowledges that it can be overridden or “upgraded.”